tsconfig.json

```diff
@@ -64,5 +61,6 @@
   },
   "include": [
     "packages/core",
+    "packages/plugin-react-native-app-state-breadcrumbs",
```

Comment:
This is one we are going to have to remember to keep up-to-date. Will look at better ways of handling this.